frameworks/hdfs/tools/docker-client/configure-environment.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export PATH=$PATH:/usr/local/sbin:/usr/sbin:/sbin
+
+sed -e "s/KDC_ADDRESS/$KDC_ADDRESS/g" -i /etc/krb5.conf
```

Comment:
I would remove the `krb5.conf` file and require that the user supply one?

Reply:
As also mentioned in #2316 we could consider using mustache here instead? Then we have consistency across how we generate these files in production.

Reply:
Yeah we could do that, just that we'd have to add bootstrap to the client bundle as well.

By production you mean our CI? We don't use this client anywhere else, right?

Reply:
By production in this case I mean the actual HDFS service. With this kind of replace, we effectively have two kinds of templating.

We would not strictly be limited to adding `bootstrap` to the client image, as we would be able to use any mustache templating library, but bootstrap would give use some other tools like DNS resolution for free.